blivet/platform.py

```diff
@@ -351,12 +352,20 @@ def setPlatformBootPartition(self):
         return [PartSpec(mountpoint="/boot", size=Size("1GiB"),
                          weight=self.weight(mountpoint="/boot"), lv=False)]
 
-    def requiredDiskLabelType(self, device_type):
-        """The required disklabel type for the specified device type."""
-        if device_type == parted.DEVICE_DASD:
```

Comment:
Since you're removing the only reason for `requiredDiskLabelType` to exist, please remove it completely -- thanks for removing an old platform-specific hack!